plugins/confluence-plugin/src/components/PageContent.tsx

```diff
@@ -12,16 +10,45 @@ import {
 import { getEntityYaml } from "../api/Cortex";
 import { getConfluenceDetailsFromEntity } from "../lib/parseEntity";
 
-const baseConfluenceUrl = "https://cortex-se-test.atlassian.net";
+import Instructions from "./Instructions";
 
 const PageContent: React.FC = () => {
   const [pageContent, setPageContent] = useState<string | undefined>();
   const [pageTitle, setPageTitle] = useState<string | undefined>();
   const [entityPage, setEntityPage] = useState<any | string>();
+  const [baseConfluenceUrl, setBaseConfluenceUrl] = useState<string>("");
+  const [errorStr, setErrorStr] = useState<string>("");
 
   const context = usePluginContext();
 
   useEffect(() => {
+    if (!context?.apiBaseUrl) {
+      return;
+    }
+    const getConfluencePluginConfig = async (): Promise<void> => {
```

Comment:
I think in the long run we'll want to add an abstraction into @cortexapps/plugin-core to retrieve configuration bundles, but for our PoC I think this works, just adding the comment for posterity.

Reply:
agreed, ideally we would have some boilerplate to show if various things went horribly wrong